doc/en/user/source/installation/upgrade.rst

```diff
@@ -30,6 +30,16 @@ The general GeoServer upgrade process is as follows:
 Notes on upgrading specific versions
 ------------------------------------
 
+ImageMosaic Sample Image Deserialization (GeoServer 2.26 and newer)
```

Comment:
The change was back ported to earlier geotools releases; so this is no longer accurate.